services/src/main/java/org/keycloak/protocol/saml/SamlProtocol.java

```diff
@@ -521,16 +555,6 @@ protected Response buildAuthenticatedResponse(AuthenticatedClientSessionModel cl
         }
     }
 
-    public static class ProtocolMapperProcessor<T> {
```

Comment:
I'd avoid moving this around in the file; it makes the diff harder.

Reply:
I hate auto formatting of Intelij